sdk/data/azcosmos/otel_constants.go

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/tracing"
+)
+
+const (
+	otelSpanNameCreateDatabase              = "create_database"
+	otelSpanNameReadDatabase                = "read_database"
+	otelSpanNameDeleteDatabase              = "delete_database"
+	otelSpanNameQueryDatabases              = "query_databases"
+	otelSpanNameReadThroughputDatabase      = "read_database_throughput"
+	otelSpanNameReplaceThroughputDatabase   = "replace_database_throughput"
+	otelSpanNameCreateContainer             = "create_container"
```

Comment:
check out discussion on whether we also need `create_container|database_if_exists` in the chat, also fine to do as a follow-up

Reply:
These operations don't exist in this SDK.